control/forms.py

```diff
@@ -64,39 +64,39 @@ class TimeConverter(forms.Form):
 
 
 currency_choices = [
-    ('eur', 'EUR'),
-    ('cad', 'CAD'),
-    ('hkd', 'HKD'),
-    ('isk', 'ISK'),
-    ('php', 'PHP'),
-    ('dkk', 'DKK'),
-    ('huf', 'HUF'),
-    ('czk', 'CZK'),
-    ('aud', 'AUD'),
-    ('ron', 'RON'),
-    ('sek', 'SEK'),
-    ('idr', 'IDR'),
-    ('inr', 'INR'),
-    ('brl', 'BRL'),
-    ('rub', 'RUB'),
-    ('hrk', 'HRK'),
-    ('jpy', 'JPY'),
-    ('thb', 'THB'),
-    ('chf', 'CHF'),
-    ('sgd', 'SGD'),
-    ('pln', 'PLN'),
-    ('bgn', 'BGN'),
-    ('try', 'TRY'),
-    ('cny', 'CNY'),
-    ('nok', 'NOK'),
-    ('nzd', 'NZD'),
-    ('zar', 'ZAR'),
-    ('usd', 'USD'),
-    ('mxn', 'MXN'),
-    ('ils', 'ILS'),
-    ('gbp', 'GBP'),
-    ('krw', 'KRW'),
-    ('myr', 'MYR')
+    ('eur', 'Euro'),
+    ('cad', 'Canadian Dollar'),
+    ('hkd', 'Hong Kong Dollar'),
+    ('isk', 'Icelandic Króna'),
+    ('php', 'Philippine peso'),
+    ('dkk', 'Danish Krone'),
+    ('huf', 'Hungarian Forint'),
+    ('czk', 'Czech Koruna'),
+    ('aud', 'Australian Dollar'),
+    ('ron', 'Romanian Leu'),
+    ('sek', 'Swedish Krona'),
+    ('idr', 'Indonesian Rupiah'),
+    ('inr', 'Indian Rupee'),
+    ('brl', 'Brazilian Real'),
+    ('rub', 'Russian Ruble'),
+    ('hrk', 'Croatian Kuna'),
+    ('jpy', 'Japanese Yen'),
+    ('thb', 'Thai Baht'),
+    ('chf', 'Swiss Franc'),
+    ('sgd', 'Singapore Dollar'),
+    ('pln', 'Poland złoty'),
+    ('bgn', 'Bulgarian Lev'),
+    ('try', 'Turkish lira'),
+    ('cny', 'Chinese Yuan'),
+    ('nok', 'Norwegian Krone'),
+    ('nzd', 'New Zealand Dollar'),
+    ('zar', 'South African Rand'),
+    ('usd', 'United States Dollar'),
+    ('mxn', 'Mexican Peso'),
+    ('ils', 'Israeli New Shekel'),
+    ('gbp', 'Pound sterling'),
+    ('krw', 'South Korean Won'),
+    ('myr', 'Malaysian Ringgit')
```

Comment:
This belongs to a different pull request

Reply:
Hardcoding is never a good practice anyway, I would suggest putting this in a model, in a database. You might be able to use an existing one and store them there, look at: https://github.com/panosl/django-currencies/

Reply:
ok, I wasn't really sure if this was hard code or not I'll have a look, I've pushed it in the same PR because it was related to currency_converter, but I can always open a different branch.